python-threatexchange/threatexchange/signal_type/pdq/pdq_hasher.py

```diff
@@ -50,8 +50,13 @@ def _convert_image_to_correct_array_dimension(image: Image.Image) -> np.ndarray:
     """
     Handle possible image format conversion or
     """
-    if image.mode == "LA" or image.mode == "I;16":
+    if (
+        image.mode == "LA"
+        or image.mode == "I;16"
+        or (image.format == "GIF" and not getattr(image, "is_animated", False))
+    ):
         # LA images (luminance with alpha) return 3 dimensional ndarray
+        # For GIF converts the first frame of a static GIF to RGB
```

Comment:
blocking question: What is the behavior if you don't convert it?

Reply:
Actually it works perfectly fine without converting. I had initially started to make these changes because the quality of the hash was poor but that was due to the solid color gifs I was using not because of the RGB convert I will change it back. Thank you for catching that.